esp/services/ws_ecl/ws_ecl_service.hpp

```diff
@@ -200,6 +203,7 @@ class CWsEclBinding : public CHttpSoapBinding
 
     int getXsdDefinition(IEspContext &context, CHttpRequest *request, StringBuffer &content, WsEclWuInfo &wsinfo);
     bool getSchema(StringBuffer& schema, IEspContext &ctx, CHttpRequest* req, WsEclWuInfo &wsinfo) ;
```

Comment:
I would consider renaming this method and moving it to be a protected member, unless there is another component that is calling it.